globalnoc-tsds-datasource/src/plugin.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "datasource",
-  "name": "GlobalNOC TSDS",
-  "id": "globalnoc-tsds-datasource",
+  "name": "GlobalNOC TSDS-Test",
+  "id": "globalnoc-tsds-datasource-test",
```

Comment:
Please don't make commits like this. I suspect you're doing "git commit -a" or something to that effect, but this is going to change the actual build/install properties of the package.